### PR TITLE
Ported ColorWidget to Fluent UI theme

### PR DESCRIPTION
**Pull request by sramaswami11**
### Reasons for making this change

[Please describe them here]
image

If this is related to existing tickets, include links to them as well.

### Checklist

* [ ] **I'm updating documentation**
  - [ ] I've [checked the rendering](https://react-jsonschema-form.readthedocs.io/en/latest/#contributing) of the Markdown text I've added
* [ ] **I'm adding or updating code**
  - [ ] I've added and/or updated tests
  - [ ] I've updated [docs](https://react-jsonschema-form.readthedocs.io/) if needed
* [ ] **I'm adding a new feature**
  - [ ] I've updated the playground with an example use of the feature

### CODE REVIEW

File: packages/fluent-ui/src/ColorWidget/ColorWidget.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import _pick from "lodash/pick";
+import { useConstCallback } from "@uifabric/react-hooks";
+
+const colorPickerStyles: Partial<IColorPickerStyles> = {
+  panel: { padding: 12 },
+  root: {
+    maxWidth: 352,
+    minWidth: 352,
+  },
+  colorRectangle: { height: 268 },
```
Comment (epicfaace) [inline packages/fluent-ui/src/ColorWidget/ColorWidget.tsx]: remove hardcoding

File: packages/fluent-ui/src/ColorWidget/ColorWidget.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  "alphaType",
+  "alphaSliderHidden",
+  "hexLabel",
+  "redLabel",
+  "greenLabel",
+  "blueLabel",
+  "alphaLabel",
+  "className",
+  "theme",
+  "styles",
```
Comment (epicfaace) [inline packages/fluent-ui/src/ColorWidget/ColorWidget.tsx:37]:
```suggestion
  "styles",
  "showPreview",
```

File: packages/fluent-ui/src/ColorWidget/ColorWidget.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  fontSize: 12,
+  fontWeight: "normal" as any,
+  fontFamily: `"Segoe UI", "Segoe UI Web (West European)", "Segoe UI", -apple-system, BlinkMacSystemFont, Roboto, "Helvetica Neue", sans-serif;`,
+};
+
+// Keys of IColorPickerProps from @fluentui/react
+const allowedProps = [
+  "componentRef",
+  "color",
+  "strings",
```
Comment (epicfaace) [inline packages/fluent-ui/src/ColorWidget/ColorWidget.tsx:26]:
```suggestion
  "strings",
  "onChange",
```

File: packages/fluent-ui/src/ColorWidget/ColorWidget.tsx (hunk 1 above)
Comment (sramaswami11) [reply packages/fluent-ui/src/ColorWidget/ColorWidget.tsx]: Done.

File: packages/fluent-ui/src/ColorWidget/ColorWidget.tsx (hunk 2 above)
Comment (sramaswami11) [reply packages/fluent-ui/src/ColorWidget/ColorWidget.tsx:37]: Done

File: packages/fluent-ui/src/ColorWidget/ColorWidget.tsx (hunk 3 above)
Comment (sramaswami11) [reply packages/fluent-ui/src/ColorWidget/ColorWidget.tsx:26]: Done.

**Review by epicfaace — APPROVED** [review]

**Pull request merged**